src/test/java/com/exasol/adapter/dialects/scalarfunction/ScalarFunctionsAbstractIT.java

```diff
@@ -332,9 +330,11 @@ void afterAll() {
         @ParameterizedTest
         @CsvSource({ "ADD, +, 4", "SUB, -, 0", "MULT, *, 4", "FLOAT_DIV, /, 1" })
         void testSimpleArithmeticFunctions(final ScalarFunctionCapability function, final String operator,
-                final int expectedResult) {
-            if (isTestDisabled(function))
+                final int expectedResult) throws SQLException {
+            if (isTestDisabled(function)) {
                 return;
+            }
+            this.valueSetter.setValueOfSingleRow(2);
```

Comment:
I find a bit confusing and hard to follow the concept the `valueSetter`, as it is an object provided by the `VirtualSchemaTable` to manipulate it.
As I understand, the `valueSetter` is attached to a table, and its responsibility is to restart the table with a single row, meaning basically `truncate` it and then `insert` a single row.

I would prefer to add more meaningful behavior to the `VirtualSchemaTable` by adding this functionality to it, and make the `valueSetter` an implementation detail encapsulated in the `VirtualSchemaTable`.